indico/modules/events/templates/display/indico/meeting.html

```diff
@@ -5,9 +5,13 @@
 
 {{ render_event_header_msg(event) }}
 
+{% set has_subheader = event.description or files or folders or lectures or conf.getOrgText()
```

Comment:
I would love to get rid of this condition and only use CSS to fix the current problem of displaying the `.event-sub-header`. Unfortunately, this is required cause there is no other way to add rounder borders to the `.event-header`.